src/shared/bus-unit-util.c

```diff
@@ -2142,6 +2142,10 @@ static int bus_append_scope_property(sd_bus_message *m, const char *field, const
         if (STR_IN_SET(field, "User", "Group"))
                 return bus_append_string(m, field, eq);
 
+        if (streq(field, "OOMPolicy")) {
```

Comment:
nit: you can add `OOMPolicy` to the end of the STR_IN_SET macro above since they return the same thing

Reply:
Do you really want to? [These lines above](https://github.com/systemd/systemd/pull/25385/files/3c83984e3f366ef8d604a6ad09ad82112f08663c#diff-67cfddb9602e31232220c4c28dcc990d00fe78d33ae49be3c9fb7baefff8ce03R2131-R2138) could use STR_IN_SET too, but they aren't.

Reply:
Yea I noticed that and was going to update it outside of this PR. I think it would make it more stylistically consistent with the other parts of this file. You're welcome to update both parts to STR_IN_SET if you're inclined

Reply:
If you plan to clean it up in a style commit I'll just leave it as it is here, for clarity.